docs/src/pages/components/slider/MusicPlayerSlider.tsx

```diff
@@ -0,0 +1,218 @@
+          sx={{
+            color: theme.palette.mode === 'dark' ? '#fff' : 'rgba(0,0,0,0.87)',
+            height: 4,
+            '& .MuiSlider-thumb': {
+              width: 8,
+              height: 8,
+              transition: '0.3s cubic-bezier(.47,1.64,.41,.8)',
+              '&:before': {
+                boxShadow: '0 2px 12px 0 rgba(0,0,0,0.4)',
+              },
+              '&:hover, &.Mui-focusVisible': {
+                boxShadow: `0px 0px 0px 8px ${
+                  theme.palette.mode === 'dark'
+                    ? 'rgb(255 255 255 / 16%)'
+                    : 'rgb(0 0 0 / 16%)'
+                }`,
+              },
+              '&.Mui-active': {
+                width: 20,
+                height: 20,
+              },
+            },
+            '& .MuiSlider-rail': {
+              opacity: 0.28,
+            },
+          }}
+        />
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            mt: -2,
+          }}
+        >
+          <TinyText>{formatDuration(position)}</TinyText>
+          <TinyText>-{formatDuration(duration - position)}</TinyText>
+        </Box>
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+            mt: -1,
+          }}
+        >
+          <IconButton aria-label="previous song">
+            <FastRewindRounded fontSize="large" htmlColor={mainIconColor} />
+          </IconButton>
+          <IconButton
+            aria-label={paused ? 'play' : 'pause'}
+            onClick={() => setPaused(!paused)}
+          >
+            {paused ? (
+              <PlayArrowRounded
+                sx={{ fontSize: '3rem' }}
+                htmlColor={mainIconColor}
+              />
+            ) : (
+              <PauseRounded sx={{ fontSize: '3rem' }} htmlColor={mainIconColor} />
+            )}
+          </IconButton>
+          <IconButton aria-label="next song">
+            <FastForwardRounded fontSize="large" htmlColor={mainIconColor} />
+          </IconButton>
+        </Box>
+        <Stack spacing={2} direction="row" sx={{ mb: 1, px: 1 }} alignItems="center">
+          <VolumeDownRounded htmlColor={lightIconColor} />
+          <Slider
+            aria-label="Volume"
+            defaultValue={30}
+            sx={{
+              color: theme.palette.mode === 'dark' ? '#fff' : 'rgba(0,0,0,0.87)',
+              '& .MuiSlider-track': {
+                border: 'none',
+              },
+              '& .MuiSlider-thumb': {
+                width: 24,
+                height: 24,
+                backgroundColor: '#fff',
+                '&:before': {
+                  boxShadow: '0 4px 8px rgba(0,0,0,0.4)',
+                },
+                '&:hover, &.Mui-focusVisible': {
+                  boxShadow: 'none',
+                },
+              },
+            }}
+          />
```

Comment:
We might miss something: we can trigger a ripple by clicking

<img width="191" alt="Capture d’écran 2021-06-16 à 00 13 15" src="https://user-images.githubusercontent.com/3165635/122130395-d52dbe00-ce37-11eb-988b-9062681972ca.png">

Reply:
Nice catch, it is from `.Mui-active`. fixed 👍